app/controllers/site_controller.rb

```diff
@@ -666,7 +666,10 @@ def set_firecloud_permissions(study_detached)
       if api_status.is_a?(Hash)
         system_status = api_status['systems']
         sam_ok = system_status.dig(FireCloudClient::SAM_SERVICE, 'ok') == true # do equality check in case 'ok' node isn't present
-        rawls_ok = system_status.dig(FireCloudClient::RAWLS_SERVICE, 'ok') == true
+        #  2025-01-31 quick and dirty workaround for false negative Rawls availability issue
```

Comment:
The current phrasing is subtly but importantly incorrect.  For reference:
* False positive: non-problem reported as problem
* False negative: problem reported as non-problem
* True positive: problem reported as problem
* True negative: non-problem reported as non-problem

The issue is that Rawls is effectively OK but is reported as not-OK, so the status report is a false positive.

```suggestion
        #  2025-01-31 quick and dirty workaround for false positive Rawls availability issue
```

Reply:
Thanks @eweitz. I always think this through the wrong way -  ie. Rawls was reporting it was "not OK" (ie. a negative that was incorrect)and so I end up using the wrong term.  Updated in `1cbcc606f`.